gametest/build.gradle

```diff
@@ -0,0 +1,75 @@
+plugins {
+    id 'java'
+    id 'gametestable'
+    alias rootLibs.plugins.quilt.loom
+}
+
+loom {
+    runs {
+        gametest {
+            name 'Game Test'
+            source sourceSets.main
+
+            server()
+
+            ideConfigGenerated(true)
+        }
+    }
+}
+
+repositories {
+    maven {
+        name = 'gametest'
+        url = '../build/gametestable'
+    }
+}
+
+dependencies {
+    minecraft(rootLibs.minecraft)
+    mappings "org.quiltmc:quilt-mappings:${gametestable.version}:intermediary-v2"
+
+    modImplementation rootLibs.quilt.loader
+}
+
+processResources {
+    final expanding = [
+        version: gametestable.version,
+        group: 'org.quilt.internal',
+        minecraft_version: rootLibs.versions.minecraft.get(),
+        quilt_loader_version: rootLibs.versions.quilt.loader.get(),
+    ]
+
+    inputs.properties(expanding)
+
+    filesMatching('quilt.mod.json') {
+        expand expanding
+    }
+}
+
+// mod test data isn't loaded without fapi, so copy it to a datapack
```

Comment:
Just pointing this out to make sure we're ok with this additional jank.

This is easier and hopefully more maintainable than a mixin to load it from resources.